tetrateci/create_istio_release.sh

```diff
@@ -16,6 +16,8 @@ else
 fi
 
 ## Set up release-builder
+go install sigs.k8s.io/bom/cmd/bom@latest
```

Comment:
Shall we use a fixed version? e.g. `v0.2.2` is the most recent release for now.

This way, the release process is more reproducible, won't see failures caused by bom versions differ at different time.

Reply:
good point, added fixed latest version (v0.2.2) of bom.